### PR TITLE
Update to align with the form

### PR DESCRIPTION
**Pull request by Lucyeoh**

### CODE REVIEW

File: standard-questions.md [hunk 1]
```diff
@@ -73,9 +74,9 @@ This document contains the set of questions that are being asked to assess each
             <td valign="top">Is there a mechanism for extracting or importing non PII data from the system in a non-proprietary format?</td>
             <td valign="top">
                 <ul>
-                    <li>Does this project have non-PII data?</li>
-                    <li>Is there a mechanism for extracting or importing non PII data from the system in a non-proprietary format?</li>
-                    <li>If yes - please describe the mechanism for extracting or importing data</li>
+                    <li>Does this project collect or use non-personally identifiable information (PII) data? </li>
```
Comment (lacabra) [inline standard-questions.md:77]: This should be abbreviated as `(non-PII)`, otherwise it can be confusing

File: standard-questions.md [hunk 2]
```diff
@@ -73,9 +74,9 @@ This document contains the set of questions that are being asked to assess each
...
+                    <li>If yes - is there a mechanism for extracting or importing non PII data from the system in a non-proprietary format?</li>
```
Comment (lacabra) [inline standard-questions.md:78]: Same here, please refer to `non-PII` (hyphenated for consistency with the above)

File: standard-questions.md [hunk 3]
```diff
@@ -73,9 +74,9 @@ This document contains the set of questions that are being asked to assess each
...
+                    <li>If yes - Describe the mechanism for extracting or importing non personally identifiable data from the system in a non-proprietary format:</li>
```
Comment (lacabra) [inline standard-questions.md:79]: change `data` for `information` for consistency sake

**Review by lacabra — CHANGES_REQUESTED** [review]

**Review by nathanbaleeta — APPROVED** [review]

File: standard-questions.md (hunk 1 above)
Comment (lacabra) [reply standard-questions.md:77]: Edited in 7e7156d

File: standard-questions.md (hunk 2 above)
Comment (lacabra) [reply standard-questions.md:78]: Edited in 7e7156d

File: standard-questions.md (hunk 3 above)
Comment (lacabra) [reply standard-questions.md]: Edited in 7e7156d

**Review by lacabra — APPROVED** [review]

**Pull request merged**